gitman/models/config.py

```diff
@@ -29,22 +30,45 @@ def __init__(self, root, filename="gitman.yml", location="gitman_sources"):
         self.location = location
         self.sources = []
         self.sources_locked = []
+        self.path = self.config_path
 
     @property
     def config_path(self):
         """Get the full path to the configuration file."""
         return os.path.normpath(os.path.join(self.root, self.filename))
-    path = config_path
+
+    @config_path.setter
+    def config_path(self, value):
+        """Set the full path to the configuration file."""
+        self._config_path = os.path.normpath(value)
```

Comment:
* The reasonment :
Right. 
It is in prevision of a future usage to not forget.
I prefer near complete POJO-like object to not have to work on it a later time

* The solution :
Do you really want I suppress not yet used setter ?